requirements/project-requirements.txt

```diff
@@ -54,7 +54,7 @@ pyelftools==0.29
 PyExifTool==0.5.0
 pyhashlookup==1.2.0
 pyimpfuzzy==0.5
-pymisp==2.4.168.1
+pymisp>=2.4.168.1,<2.5.0
```

Comment:
that is wrong, please revert this. Anywhere we use `<` and `>` for dependencies, why we should add that here?

It is important to pick a single version otherwise we risk unattended consequences in deployments.

we could pin the last version 2.4.172 and see if that dependency errors is solved.